### PR TITLE
Project import generated by Copybara.

### PR DESCRIPTION
**Pull request by kswang2400**
PiperOrigin-RevId: 425717115

### CODE REVIEW

File: .github/workflows/python_tools.yml [hunk 1]
```diff
@@ -51,18 +51,19 @@ jobs:
       run: |
        python -m unittest discover -p '*_test.py' --failfast -v
       working-directory: ./tools/explorer
-    #---------- Configuration scoring ----------#
```
Comment (charbull) [inline .github/workflows/python_tools.yml:54]: those are deleted

**Review by charbull — CHANGES_REQUESTED** [review]
it is deleting the scoring code.
I think you can update the copybara file to ignore the scorer for now?

File: .github/workflows/python_tools.yml (hunk 1 above)
Comment (kswang2400) [reply .github/workflows/python_tools.yml:54]: Done

**Review by trav3711 — APPROVED** [review]
I'm not sure about the scoring code but all of the changes I want are in there.

**Comment by charbull** [comment]
closing it in attempt to sync

**Pull request closed without merging**